cpp/src/groupby/sort/group_histogram.cu

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <lists/utilities.hpp>
+#include <reductions/histogram_helpers.hpp>
```

Comment:
Shall we move this file to `include`?

Reply:
Sounds good.

Reply:
How about `lists/utilities.hpp`? Both files are now used in many places so I'm not sure if we just move one.

Reply:
:+1: 

`lists/utilities.hpp` could be a separate PR (just try to make this PR as small as possible)